phy/cluster/manual/clustering.py

```diff
@@ -8,12 +8,13 @@
 
 import numpy as np
 
-from ...utils.array import _as_array, _is_array_like
-from ._utils import (_unique,
-                     _spikes_in_clusters,
-                     _spikes_per_cluster,
-                     UpdateInfo,
-                     )
+from ...utils import (_as_array,
+                      _is_array_like,
+                      _unique,
+                      _spikes_in_clusters,
+                      _spikes_per_cluster,
+                      )
+from ._utils import UpdateInfo
```

Comment:
I know we've been through this before, but this is why this kind of relative import is really confusing!